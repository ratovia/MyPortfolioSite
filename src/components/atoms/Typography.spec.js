import React from "react"
import renderer from "react-test-renderer"
import {
  Typography,
  TypographyH4,
  TypographyH5,
  TypographyH6,
} from "./Typography"
describe("Typography", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Typography label="Hello World" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it("renders correctly", () => {
    const tree = renderer
      .create(<Typography label="Hello World" color="red" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
describe("TypographyH4", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<TypographyH4 label="Hello World" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it("renders correctly", () => {
    const tree = renderer
      .create(<TypographyH4 label="Hello World" color="red" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
describe("TypographyH5", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<TypographyH5 label="Hello World" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it("renders correctly", () => {
    const tree = renderer
      .create(<TypographyH5 label="Hello World" color="red" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
describe("TypographyH6", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<TypographyH6 label="Hello World" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it("renders correctly", () => {
    const tree = renderer
      .create(<TypographyH6 label="Hello World" color="red" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
