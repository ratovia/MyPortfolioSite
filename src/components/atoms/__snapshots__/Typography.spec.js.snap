// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography renders correctly 1`] = `
<p
  className="sc-bdvvaa lIoxt"
>
  Hello World
</p>
`;

exports[`Typography renders correctly 2`] = `
<p
  className="sc-bdvvaa eTjOMu"
  color="red"
>
  Hello World
</p>
`;

exports[`TypographyH4 renders correctly 1`] = `
<h4
  className="sc-gsDJrp gKGDLI"
>
  <div>
    <p
      className="sc-bdvvaa lIoxt"
    >
      Hello World
    </p>
  </div>
</h4>
`;

exports[`TypographyH4 renders correctly 2`] = `
<h4
  className="sc-gsDJrp gKGDLI"
>
  <div>
    <p
      className="sc-bdvvaa eTjOMu"
      color="red"
    >
      Hello World
    </p>
  </div>
</h4>
`;

exports[`TypographyH5 renders correctly 1`] = `
<h5
  className="sc-dkPtyc liUvOS"
>
  <p
    className="sc-bdvvaa lIoxt"
  >
    Hello World
  </p>
</h5>
`;

exports[`TypographyH5 renders correctly 2`] = `
<h5
  className="sc-dkPtyc liUvOS"
>
  <p
    className="sc-bdvvaa eTjOMu"
    color="red"
  >
    Hello World
  </p>
</h5>
`;

exports[`TypographyH6 renders correctly 1`] = `
<h6
  className="sc-hKwCoD hglaFY"
>
  <p
    className="sc-bdvvaa lIoxt"
  >
    Hello World
  </p>
</h6>
`;

exports[`TypographyH6 renders correctly 2`] = `
<h6
  className="sc-hKwCoD hglaFY"
>
  <p
    className="sc-bdvvaa eTjOMu"
    color="red"
  >
    Hello World
  </p>
</h6>
`;
